### PR TITLE
feat(toolbar): added width control

### LINKED ISSUE #3449: Toolbar filters are narrower in Chrome
Toolbar filters are narrower in Chrome than either Safari or Firefox. This becomes obvious when using long placeholder text, which forces an ellipsis. I was able to replicate the issue here: https://www.patternfly.org/v4/documentation/core/components/toolbar#toggle-group , using placeholder="Filter by exact username" 

Here's an example from the RBAC UI:
Screen Shot 2020-09-03 at 2 08 04 PM

### LINKED ISSUE #3894: Search Input used in Toolbar will shift content to the right.
If you use the search input component within a toolbar as shown in the following example.  It will push over the content when you type in it.  The team was expecting the content in the toolbar to not move to the right.

https://codesandbox.io/s/searchinputintoolbar-19538

CC: @srambach

### PR DESCRIPTION
**Pull request by mattnolting**
closes #3894 
closes #3449 
Added width control to and `toolbar item`.

### CODE REVIEW

**Review by mcarrano — COMMENTED** [review]
Looks OK to me, but I'll defer to others for approval.

File: src/patternfly/components/Toolbar/examples/Toolbar.md [hunk 1]
```diff
@@ -210,6 +210,37 @@ In some instances, it may be necessary to adjust spacing explicitly where items
...
+    {{/toolbar-content-section}}
+  {{/toolbar-content}}
+{{/toolbar}}
+```
+
+### Width control
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `--pf-c-toolbar__item--Width{-on-[breakpoint]}: {width}` | `.pf-c-toolbar__item` |  Modifies the width value of a `toolbar item`. |
```
Comment (mcoker) [inline src/patternfly/components/Toolbar/examples/Toolbar.md:242]:
```suggestion
| `--pf-c-toolbar__item--Width{-on-[breakpoint]}: {width}` | `.pf-c-toolbar__item` |  Modifies the width value of a toolbar item. |
```

File: src/patternfly/components/InputGroup/examples/InputGroup.md [hunk 2]
```diff
@@ -133,3 +148,4 @@ When using the `.pf-c-input-group` always ensure labels are used outside the inp
 | `.pf-c-input-group` | `<div>` |  Initiates the input group. **Required** |
 | `.pf-c-input-group__text` | `<span>` |  Initiates the input group text. This can be used to show text, radio, icons, or check boxes. |
 | `.pf-m-plain` | `.pf-c-input-group__text` | Removes the border from the text element. |
+| `--pf-c-input-group--Width{-on-[breakpoint]}: {width}` | `.pf-c-input-group` |  Modifies the width value of an `input group`. |
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/examples/InputGroup.md:151]:
```suggestion
| `--pf-c-input-group--Width{-on-[breakpoint]}: {width}` | `.pf-c-input-group` |  Modifies the width value of an input group. |
```

File: src/patternfly/components/InputGroup/examples/InputGroup.md [hunk 3]
```diff
@@ -116,6 +116,21 @@ cssPrefix: pf-c-input-group
 {{/input-group}}
 ```
 
+### Width control
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/examples/InputGroup.md:119]: I'm wondering if we need this for an input group, or if we could just enable it on a toolbar item? Was there a reason we need it on an input group, too?

**Review by mcoker — CHANGES_REQUESTED** [review]

**Comment by mattnolting** [comment]
@mcoker updated

File: src/patternfly/components/InputGroup/examples/InputGroup.md (hunk 3 above)
Comment (mattnolting) [reply src/patternfly/components/InputGroup/examples/InputGroup.md]: My reasoning was that the same issue would occur outside of a `toolbar`. But we can limit this PR to `toolbar`

**Review by mcoker — APPROVED** [review]
L🏆TM!!

**Pull request merged**